gen/aircraft.py

```diff
@@ -272,6 +273,10 @@ def client_units(self) -> List[FlyingUnit]:
     def aircraft_type(self) -> FlyingType:
         """Returns the type of aircraft in this flight."""
         return self.units[0].unit_type
+    
+    @property
+    def departure_delay_delta(self) -> timedelta:
+        return timedelta(seconds=self.departure_delay)
```

Comment:
Good follow up cleanup would be to change the type of the property. I just replaced a whole bunch of ints with timedeltas, but seems I missed this one.